scripts/pr_reviewer.py

```
@@ -1,104 +1,188 @@
+            if status == "✅":
+                status = "⚠️"
+            invalid_found = True
+        commit_results.append(f"| `{sha}` | {status} | {msg} | {'; '.join(issues) if issues else 'OK'} |")
+    result = ["### Commit Message Validation\n", "| Commit | Status | Message | Notes |", "|--------|--------|---------|-------|"]
+    result.extend(commit_results)
+    return "\n".join(result), not invalid_found
+
+def summarize_pr(pr, gitleaks_issues=None, compliance_issues=None):
+    summary = f"### High-level PR Summary\n**Files changed:** {pr.changed_files}\n**Lines added:** {pr.additions}\n**Lines removed:** {pr.deletions}\n**Commits:** {pr.commits}\n"
+    risky_files = set()
+    if gitleaks_issues: risky_files.update(gitleaks_issues.keys())
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)